### ISSUE TITLE
Duplicate categories

### ISSUE THREAD

**Issue by Sicmatr1x** [TriDiamond/hexo-theme-obsidian#12]
If you have these two markdown files, you will have four categories in category page. Because order of the categorny "后端" is not same in the two markdown files.

```
title: A
categories:
    - 后端
    - Java
```

```
title: B
categories:
    - Node.js
    - 后端
```

**Comment by TriDiamond** [comment]
This is interesting, because categories listed in you post file, the order you put it will generate leveled categories. So basically you first backend category is a first level category, in your second example the backend category is a second level category. My category page did I not separate leveled categories that’s why you see two backend categories at the same time.

However I’m planning to reshape the categories page in my 1.4.0 release, which include a muti-categorized view. That way this type of misunderstanding won’t exist.

**Comment by Sicmatr1x** [comment]
like this
![微信截图_20191101165220](https://user-images.githubusercontent.com/26240752/68013497-0cce0d80-fcc8-11e9-8403-79a0efef875a.png)

**Comment by Sicmatr1x** [comment]
Yes, I think the categories should like extend father class and tags like implements interface in Java or other OOP langtuage. I know my use of categories is not very good, but I can't help but add multiple categories.

**Comment by TriDiamond** [comment]
> Yes, I think the categories should like extend father class and tags like implements interface in Java or other OOP langtuage. I know my use of categories is not very good, but I can't help but add multiple categories.

Yes I will work on categories and tags on my coming up releases.

**Comment by TriDiamond** [comment]
@Sicmatr1x The multi-level categories support is released in [`v1.4.0`](https://github.com/TriDiamond/hexo-theme-obsidian/releases/tag/v1.4.0)

Hope you like it! ❤️
